netebpfext/net_ebpf_ext_hook_provider.c

```diff
@@ -41,6 +43,14 @@ typedef struct _net_ebpf_extension_hook_provider
         LIST_ENTRY filter_context_list; ///< Linked list of filter contexts that are attached to this provider.
 } net_ebpf_extension_hook_provider_t;
 
+typedef struct _net_ebpf_extension_program_invoke_parameters
+{
+    net_ebpf_extension_wfp_filter_context_t* filter_context;
+    void* program_context;
+    uint32_t verdict;
+    ebpf_result_t result;
+} net_ebpf_extension_program_invoke_parameters_t;
```

Comment:
```suggestion
typedef struct _net_ebpf_extension_invoke_programs_parameters
{
    net_ebpf_extension_wfp_filter_context_t* filter_context;
    void* program_context;
    uint32_t verdict;
    ebpf_result_t result;
} net_ebpf_extension_invoke_programs_parameters_t;
```
for consistency with the function names

Reply:
Updated